packages/atomic/src/components/atomic-breadcrumb-manager/atomic-breadcrumb-manager.tsx

```diff
@@ -41,153 +48,160 @@ export class AtomicBreadcrumbManager implements InitializableComponent {
   @State() private collapsedBreadcrumbsState: string[] = [];
   @State() public error!: Error;
 
+  /**
+   * Number of breadcrumbs to be shown before collapsing.
+   */
   @Prop() public collapseThreshold = 5;
+  /**
+   * Character that divides each path segment in a category facet breadcrumb
+   */
   @Prop() public categoryDivider = '/';
 
+  @BindStateToI18n()
+  @State()
+  public strings: I18nState = {
+    breadcrumb: (variables) =>
+      this.bindings.i18n.t('removeFilterOn', variables),
+  };
+
   public initialize() {
     this.breadcrumbManager = buildBreadcrumbManager(this.bindings.engine);
   }
 
-  private get facetBreadcrumbs() {
-    return this.breadcrumbManagerState.facetBreadcrumbs.map((breadcrumb) => {
-      const breadcrumbsValues = this.getBreadrumbValues(breadcrumb);
-      return (
-        <ul part="breadcrumbs">
-          <li part="breadcrumb-label">{breadcrumb.field}:&nbsp;</li>
-          {breadcrumbsValues}
-        </ul>
-      );
-    });
+  private getBreadcrumbValue(
+    value: string,
+    breadcrumbValue: BreadcrumbValue<BaseFacetValue> | CategoryFacetBreadcrumb
+  ) {
+    return (
+      <button
+        part="breadcrumb-button"
+        class="text-on-background-variant breadcrumb-button flex items-center"
+        aria-label={this.strings.breadcrumb({value})}
+        onClick={() =>
+          this.breadcrumbManager.deselectBreadcrumb(breadcrumbValue)
+        }
+      >
+        <span
+          part="breadcrumb-value-label"
+          class="whitespace-nowrap overflow-ellipsis overflow-hidden"
+        >
+          {value}
+        </span>
+        <div
+          part="breadcrumb-clear"
+          role="button"
+          class="breadcrumb-clear ml-1.5"
+          innerHTML={mainclear}
+        />
+      </button>
+    );
+  }
+
+  private getBreadcrumbWrapper(field: string, children: any) {
+    return (
+      <div class="flex items-center text-sm" part="breadcrumb-wrapper">
+        <span class="text-on-background" part="breadcrumb-label">
+          {facetStore.get(field) || field}:&nbsp;
+        </span>
+        {children}
+      </div>
+    );
   }
 
-  private getBreadrumbValues(breadcrumb: Breadcrumb<FacetValue>) {
+  private getBreadcrumbValues(
+    breadcrumb: Breadcrumb<BaseFacetValue & {value: string}>
```

Comment:
I would encourage simplifying the types by moving away from generic and intersection types. If this function should accept any kind of breadcrumb, then let us write that out:

```
breadcrumb: FacetBreadcrumb | DateFacetBreadcrumb | NumericFacetBreadcrumb | CategoryFacetBreadcrumb
```
If this sequence recurs, then we can store the expression in a type alias

```
type AnyBreadcrumb = FacetBreadcrumb | DateFacetBreadcrumb | NumericFacetBreadcrumb | CategoryFacetBreadcrumb

breadcrumb: AnyBreadcrumb 
```

Reply:
I guess to clarify here, `value` in this case actually refers to something outside the usual FacetValue, maybe `preprocessedValue` or `valueToDisplay` would be a better name. Basically it's just the value that should be displayed vs the raw value for the facet. For regular facet breadcrumbs right now the rawValue is equal to the value to display but for range facets its not.

So I think in this case an intersection type makes sense because we want a generic facet value + an additional attribute outside the facet value.